examples/scaling-criteo/04-Triton-Inference-with-Merlin-Models-TensorFlow.ipynb

```diff
@@ -74,7 +74,6 @@
     "\n",
     "import tritonclient.grpc as grpcclient\n",
     "from nvtabular.workflow import Workflow\n",
-    "import merlin.models.tf as mm\n",
```

Comment:
This is your problem. You have to import this. Even though we dont use it in the notebook it loads some required artifacts that allow tensorflow models to run the way we expect.

Reply:
Subtle.  I blindly followed the advice of flake8.  I added a comment to explain it's purpose and a `# noqa` so that pre-commit could run happily.